source/atlas-cli-deploy-docker.txt

```diff
@@ -167,6 +167,124 @@ deployment with `Docker Compose <https://docs.docker.com/compose/>`__.
 
          mongosh mongodb://root:root@localhost:27778/?directConnection=true
 
+Persist Data Across Runs with Docker Compose
+--------------------------------------------
+
+You can persist data across multiple runs with Docker Compose.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Create a script to start and stop your local deployment.
```

Comment:
I was testing this process and created this script and ran the command to make it executable by me, but it doesn't seem we actually have them run the script in this procedure. When I tried to follow the procedure to Create a Local Deployment with Docker Compose, I got the error "Failed to execute script docker-compose" (filed a separate ticket for that: https://jira.mongodb.org/browse/DOCSP-37759)... as a result I could not test the rest of this tutorial.

Does Docker run it automatically if you have Docker as a prereq? Do you need to save the file in a specific location to do that?

Reply:
@mmarcon, can you clarify how the script the works? Do we need a step for manually running it?

Reply:
@sarahsimpers you don't run the script. It is mounted inside the container and it is run automatically when the Docker container is started (see the related entry in the `docker-compose.yaml` file).

In terms of prereqs, you need to have Docker installed. [Here](https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-deploy-docker/#create-a-local-atlas-deployment-with-docker-compose) we also suggest having `docker-compose` installed, which is fine. But for a while, I believe that hasn't been necessary as you can also do `docker compose up` (`compose` as a subcommand of `docker` instead of its own command).

Reply:
@mmarcon, do users also need to pull down the latest `mongodb/atlas` docker image like they do in the first procedure?

Reply:
@mmarcon confirmed via Slack: "no. When you do docker compose up , the image is pulled down automatically from docker hub."